tests/rptest/tests/retention_policy_test.py

```diff
@@ -97,6 +97,7 @@ def done():
                    backoff_sec=5,
                    err_msg="Segments were not removed")
 
+    @ignore  # https://github.com/vectorizedio/redpanda/issues/2406
```

Comment:
https://github.com/vectorizedio/redpanda/pull/2476

Reply:
Just merged that and removing the disabling commit here.

Reply:
I don't think it worked.

Reply:
You're right, I need more coffee.